conf/gulp-tasks/library.gulpfile.js

```diff
@@ -1,46 +1,90 @@
 const { series, parallel, src, dest, watch } = require('gulp');
-
-const rename = require('gulp-rename');
-const sass = require('gulp-sass');
+const path = require('path');
 const postcss = require('gulp-postcss');
-const uglify = require('gulp-uglify-es').default;
+const sass = require('gulp-sass');
 
 const getLibraryVersion = require('./utils/libversion');
 const { BundleType, BundleTaskFactory } = require('./bundle/bundletaskfactory');
+const { DEV_LOCALE, BUILD_LOCALES } = require('../i18n/constants');
+const LocalFileParser = require('../i18n/localfileparser');
+const MinifyTaskFactory = require('./bundle/minifytaskfactory');
+const TranslationResolver = require('../i18n/translationresolver');
+const Translator = require('../i18n/translator');
 
-const bundleTaskFactory = new BundleTaskFactory(getLibraryVersion());
-
-function bundle () {
-  return bundleTaskFactory.create(BundleType.MODERN)();
-}
+/**
+ * Creates the un-minified legacy JS bundle, compiles CSS, and kicks off the watch task.
+ * @returns {Promise<Function>}
+ */
+exports.dev = function devJSBundle () {
```

Comment:
I think npm run dev still isn't quite functional but besides that everything lgtm!